### PR TITLE
Storing invoices at gdrive

### PR DESCRIPTION
**Pull request by BognaLew**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -130,22 +122,14 @@ def order_subscription():
...
 
     if form.validate_on_submit():
-        user = db.get_client_by_email(form.email.data)
-        user.bundles.append(BundleInfo(email=form.email.data,
-                                       name=form.packets.data,
-                                       date_from=datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"),
-                                       date_to=(datetime.datetime.now() + datetime.timedelta(days=manifest.PACKETS[form.packets.data]['duration'])).strftime("%Y-%m-%d %H:%M:%S")
-                                       ))
-        db.serialize(user)
+        # place for processing from data
```
Comment (Kaszana102) [inline src/main.py]: why removed ordering packets?

Comment (BognaLew) [reply src/main.py]: I haven't noticed it during fixing merge conflicts but it should be alright now

Comment (Kaszana102) [reply src/main.py]: problem solved

**Review by Kaszana102 — APPROVED** [review]
seems ok

**Review by dlydka — APPROVED** [review]

**Pull request merged**